src/components/elements/controlledDisplayFiltersGroup/index.tsx

```diff
@@ -14,18 +14,31 @@ export interface ControlledDisplayFiltersGroupProps {
   filters: DisplayFilterPartialProps[];
   toggleAllLabel?: string;
   toggleShowAll: (show: boolean) => void;
+  isToggleAll?: boolean;
```

Comment:
I'm a bit confused, what's the difference between `toggleShowAll` and `isToggleAll`?